src/components/Identification/index.js

```diff
@@ -174,7 +256,7 @@ export const Identification = props => {
    * Handles the switch of showing the default and preferred image
    */
   function handlePhotoSwitch() {
-    setSwitchPhotos(!switchPhotos);
+    setisPhotosSwitched(!isPhotosSwitched);
   }
```

Comment:
If you didn't catch it from the files above, the variable "switchPhotos" has been changed to "isPhotosSwitched" for clarity on the purpose of the variable